engine/installation/linux/suse.md

```diff
@@ -149,13 +180,25 @@ the repository.
     $ sudo zypper install docker-ee-<VERSION_STRING>
     ```
 
-    Start Docker:
+4.  Configure Docker to use the `btrfs` filesystem. Edit the file
```

Comment:
@mstanleyjones  Why do people have to configure it if it's used by default already?